src/components/negotiations-panel.tsx

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+//import DispenserItem from './dispenser-item';
+import { makeStyles } from '@material-ui/core/styles';
+import TreeView from '@material-ui/lab/TreeView';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import TreeItem from '@material-ui/lab/TreeItem';
+import { SpotlightValue } from './spotlight-value';
+import * as NegotiationStatusManager from '../negotiation-status-manager';
```

Comment:
using `import NegotiationStatusManager from '../negotiation-status-manager'` won't compile with `NegotiationStatusManager.NegotiationConflict`

Reply:
```ts
import NegotiationStatusManager from '../negotiation-status-manager'
```
this does a default import, since this module uses something other than the default import, you need to do
```ts
import { ... } from '...'
```
or, use the import * form like the one used now.

I don't think theres anything wrong with the import * form.

Reply:
Mmm .. yeah you're right in this case because we end up using the whole functionalities stored in the file. But in the case of a library (let's say Lodash), we only want to import the functionalities we want to use and not the whole library; if we import the whole library, our app bundle will end up being bigger. Webpack ends up playing its role here, Webpack is in charge of looking for the functionality we want on the library and its dependencies.  In our case, we access Webpack through react-script.

Reply:
Some interesting info
https://stackoverflow.com/questions/58741044/why-webpack-doesnt-tree-shake-the-lodash-when-using-import-as
https://stackoverflow.com/questions/42051588/wildcard-or-asterisk-vs-named-or-selective-import-es6-javascript/42051641

It appears webpack does tree-shake `import *` but only under certain conditions. I'm not sure if it works for typescript modules, I guess it probably depends on the webpack config in create-react-app.

Reply:
shorten import names 5706218